addons/skin.estuary/xml/Variables.xml

```diff
@@ -542,10 +542,10 @@
 	<variable name="BreadcrumbsGameVar">
 		<value>$LOCALIZE[15016]</value>
 	</variable>
-	<variable name="RepeatButtonColordiffuseVar">
+	<!-- <variable name="RepeatButtonColordiffuseVar">
 		<value condition="Control.HasFocus(704)">button_focus</value>
 		<value>FFFFFFFF</value>
-	</variable>
+	</variable> -->
```

Comment:
From git blame I can this came in with the orginal merge of Estuary to Master. I see that Control id 704 is used is 2 places, one of which is https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/xml/PlayerControls.xml#L121 and that has a `$VAR[PlayerControlsRepeatImageVar]` https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/xml/PlayerControls.xml#L142 so it seems that `RepeatButtonColordiffuseVar` is indeed redundant and can be removed.

Reply:
Force pushed with removal instead of commentating out

Reply:
Yep I can confirm introduction was this https://github.com/phil65/skin.estuary/commit/4c9303c8e592b38f0611a6514b7783a4c417d0f4 and https://github.com/phil65/skin.estuary/commit/8b148f7c33620afa782ea45d9da2e9fa31a98ebd with neither use applying anymore.

Reply:
I'll mark resolve, hope that's ok.